x-pack/plugins/cases/common/api/helpers.ts

```diff
@@ -82,3 +83,10 @@ export const getCaseUsersUrl = (id: string): string => {
 export const getCasesDeleteFileAttachmentsUrl = (id: string): string => {
   return INTERNAL_DELETE_FILE_ATTACHMENTS_URL.replace('{case_id}', id);
 };
+
+export const getCustomFieldUpdateUrl = (caseId: string, customFieldId: string): string => {
```

Comment:
nit: Let's stick with the replace terminology. For example, getCustomFieldReplaceUrl. Same for other occurrences in the PR.